parser/parser.c

```diff
@@ -63,6 +63,9 @@ PARSER *parser_init(RRDHOST *host, void *user, void *input, PARSER_INPUT_TYPE fl
         rc += parser_add_keyword(parser, PLUGINSD_KEYWORD_END, pluginsd_end);
         rc += parser_add_keyword(parser, PLUGINSD_KEYWORD_BEGIN, pluginsd_begin);
         rc += parser_add_keyword(parser, "SET", pluginsd_set);
+        rc += parser_add_keyword(parser, PLUGINSD_KEYWORD_GUID, pluginsd_guid);
```

Comment:
We don't need to expose these to collectors, you can call this in logfile.c:scan_metalog_file() after the call to `parser_init`.

Reply:
That was my original plan but Kosta was explicit that we should have one plugins.D language, so this one is a keyword of the language. Ideally I'd have a more optimized language for the metadata log.

Currently, the action is NULL for the plugins so it will not be parsed.

Reply:
This can be reused if we add GUID to the streaming protocol

Reply:
I don't think that Costa was explicit that it needs to go here and I do not think that we should expose it to the collectors as it adds no functionality, but adds a risk. Please move it as I have indicated.

Reply:
I will move it, but you do realize this is only possible because Stelios went above and beyond to implement pluggable language keywords.

Reply:
I do. I've already told him that I love it three times and used the same technique to keep the new replication keywords local to streaming.